provider/provider.go

```diff
@@ -356,10 +356,11 @@ func (p *dockerNativeProvider) Read(ctx context.Context, req *rpc.ReadRequest) (
 	label := fmt.Sprintf("%s.Read(%s)", p.name, urn)
 	logging.V(9).Infof("%s executing", label)
 	id := req.GetId()
+	inputs := req.GetInputs()
+	properties := req.GetProperties()
 
-	// For now, we will return the minimum implementation for Read,
-	// until we find a use case for reading an image from a registry and comparing it to existing state
-	return &rpc.ReadResponse{Id: id}, nil
+	// Return properties as passed, since we do no reconciliation,
+	return &rpc.ReadResponse{Id: id, Inputs: inputs, Properties: properties}, nil
```

Comment:
From my understanding of Read method this seems suspect. In the bridged providers, Read method performs extraction of inputs from outputs in some cases, not propagation of req.GetProperties(). Read method implements two distinct scenarios, refresh and import. IN the case of import, inputs need to be inferred from cloud state.

Reply:
Have we tested Import on this resource, or do we plan to do so. Fixing this (import) up can perhaps be out of scope of this PR but would be lovely to have a plan.

Reply:
https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pf/tfbridge/provider_read.go#L33 is the Read code I've been struggling with lately

Reply:
From chat: import makes little sense for this resource, so not to worry.